website/docs/proxy/index.md

```diff
@@ -6,12 +6,365 @@ tags: [podman-desktop, installing, windows, macos, linux]
+
+  Consider enabling the **User mode networking** option when creating your Podman Machine to route the network traffic through your VPN.
+
+- The Podman Machine connects directly to the external network.
+
+  However, a restricted environment might block all traffic no going to a proxy.
+
+  Consider configuring Podman Desktop and your Podman Machine to route the traffic through a proxy.
+
+This tutorial is guiding you through the required steps to work in a restricted environment.
+
+## Installing Podman Desktop and Podman
+
+<Tabs groupId="operating-systems">
+<TabItem value="win" label="Windows">
+
+On Windows, to install Podman Desktop and Podman in a restricted environment, consider using the installer for restricted environments.
+
+- It has all artifacts required to install Podman Desktop and Podman.
+- It does not require Internet access to download resources during installation.
+- However, it does not contain additional utilities, such as Compose or Kind.
+
+1. Check that your environment has:
+
+   - 6 GB RAM for the Podman Machine.
+   - Windows Subsystem for Linux version 2 (WSL 2) prerequisites. See [Enabling WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install), [WSL basic commands](https://learn.microsoft.com/en-us/windows/wsl/basic-commands), and [Troubleshooting WSL 2](https://learn.microsoft.com/en-us/windows/wsl/troubleshooting#error-0x80370102-the-virtual-machine-could-not-be-started-because-a-required-feature-is-not-installed):
+     - The Windows user has administrator privileges.
+     - Windows 64bit.
+     - Windows 10 Build 19043 or greater, or Windows 11.
+     - On a virtual machine: [Nested Virtualization enabled](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization#configure-nested-virtualization).
+
+1. Prepare your system.
+
+   Enable the WSL feature, without installing the default Ubuntu distribution of Linux.
+
+   Open the Command Prompt, and run:.
+
+   ```shell-session
+   > wsl --install --no-distribution
+   ```
+
+1. Restart your computer.
+
+1. Download the installer for restricted environments.
+
+   From a host that has access to external resources, go to the [Windows downloads page](/downloads/windows), and click the link for your platform: `x64` or `arm64`.
+
+1. Copy the downloaded file to the Windows host in a restricted environment.
+
+1. Install Podman Desktop.
+
+   Run the Windows installer for restricted environments.
+
+1. The **Dashboard** screen displays: _<Icon icon="fa-solid fa-info" size="lg" /> Podman needs to be set up_.
+
+   ![Podman needs set up screen](img/dashboard-podman-needs-set-up.png)
+
+   Click the **Set up** button.
+
+   Review and validate all confirmation screens to set up the Podman Machine.
+
+   <details>
+   <summary>
+
+   Optionally, when you are using a VPN, consider enabling user mode networking:
```

Comment:
I think we should be more direct

"If using a VPN, enable user mode networking".

Rather than saying "optionally" and "consider"

Reply:
That's not always necessary. You need it only if the VPN is configured to capture all the traffic that is not for localhost. (OpenShift local enables user mode networking by default, silently, and without giving the option to disable it, because it is a pain to explain).